#! /bin/sh /usr/share/dpatch/dpatch-run
## 03-link-javadoc-to-classpath.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove reference to web documentation and refer instead to 
## DP: classpath-doc.

@DPATCH@
diff -urNad libjna-java-3.0.2~/build.xml libjna-java-3.0.2/build.xml
--- libjna-java-3.0.2~/build.xml	2008-02-28 16:34:44.000000000 -0300
+++ libjna-java-3.0.2/build.xml	2008-05-20 23:09:25.000000000 -0300
@@ -450,7 +450,7 @@
       <doctitle>JNA API Documentation</doctitle>
       <header>${header}</header>
       <bottom>${footer}</bottom>
-      <link href="http://java.sun.com/j2se/1.4.2/docs/api/"/>
+      <link href="file:///usr/share/doc/classpath-doc/api/"/>
 
       <packageset dir="${src}" defaultexcludes="yes">
         <patternset>
